### PR TITLE
delete similarity func

### PR DESCRIPTION
**Pull request by yoouung**

### CODE REVIEW

File: duplication/duplication.py [hunk 1]
```diff
@@ -1,127 +1,23 @@
...
-        dr.find_element(By.CSS_SELECTOR, "body > div.layout-main > div > div > div.board > div.list_form_outer > div > form > fieldset > div > table > tbody > tr:nth-child(2) > td.extraVars.pointer.update-check").click()
-
-        # 평균 표절률 텍스트 값을 변수 rate에 저장
-        rate = dr.find_element(
-            By.CSS_SELECTOR, "body > div.layout-main > div > div > div.board > div.list_form_outer > div > div > div:nth-child(1) > div.boardReadExtraVars > table > tbody > tr:nth-child(5) > td").text
-        print("평균 표절률:", rate)
-    except:
-        print("표절 검사 실패")
-    finally:
-        dr.quit()
```
Comment (doorooful) [inline duplication/duplication.py:119]: So sad

**Review by doorooful — COMMENTED** [review]
Please refer to the comments :)

File: README.md [hunk 2]
```diff
@@ -18,9 +18,11 @@
    - Slang API(google perspective API)
      - `(.venv) % pip install google-api-python-client`
    - Duplication Detection
-     - `sample install code`
+     - `(.venv) % pip install pandas`
+     - `(.venv) % pip install konlpy`
+     - `(.venv) % pip install scikit-learn`
    - Typo
-     - ` % brew install openssl@1.1`
+     - ` % brew install b`
```
[the remark this answers is not in the document]
Comment (yoouung) [reply README.md]: I updated the code. check it please :)

**Review by doorooful — DISMISSED** [review]
Thanks :)

**Review by doorooful — APPROVED** [review]

**Pull request merged**